cumulusci/tasks/bulkdata/load.py

```diff
@@ -315,19 +320,16 @@ def _process_job_results(self, mapping, step, local_ids):
     def _generate_results_id_map(self, step, local_ids):
```

Comment:
@davidmreed : I'm inclined to change this to take just the generator instead of the whole step. Do you agree?

Reply:
My first argument is that it obscures the control flow somewhat to have get_results happen in a sub-method instead of the main method.

My second argument is that it makes clearer that the step itself is not modified or even read by the method. Just the results are used.

Reply:
@prescod  I'm fine with that change.